baseline/OSPS-QA.yaml

```diff
@@ -237,3 +227,60 @@ criteria:
     control_mappings:       
       BPB: B-G-3
     security_insights_value: # TODO
+
+  - id: OSPS-QA-11
+    maturity_level: 2
+    criterion: |
+      All released software assets MUST be
+      delivered with a software bill of materials
+      (SBOM) in SPDX or CycloneDX format accounting
+      all direct software dependencies with their
```

Comment:
Do we mean only direct dependencies here, or also transitive dependencies for this level-2 criteria?

Reply:
(We don't say anything about transitive dependencies except in the details on QA-03)

Reply:
the original QA-03 required direct and transitive. I dropped it to require only  the direct dependencies as both NTIA and BSI only require just the direct ones.